src/View/OutputView.js

```diff
@@ -0,0 +1,25 @@
+const { Console } = require('@woowacourse/mission-utils');
+const { MESSAGE, RESULT } = require('../Util/Constants');
+
+const OutputView = {
+  printStart() {
+    Console.print(MESSAGE.START);
+  },
+
+  printResult({ strike, ball }) {
+    const strikeResult = strike ? strike + RESULT.STRIKE : '';
+    const ballResult = ball ? ball + RESULT.BALL + RESULT.BLANK : '';
+
+    if (strike || ball) {
+      return Console.print(ballResult + strikeResult);
+    }
```

Comment:
그리고 이렇게 로직을 짰을 땐 ball만 나왔을 경우 '2볼'이 아니라 '2볼 '로 나오게 되는데, 이런 상황에선 trim() 메소드를 활용해서 정확한 값을 리턴하는 게 좋아보입니다👍👍

```suggestion
    if (strike || ball) {
      return Console.print(`${ballResult}${strikeResult}`.trim());
    }
```